main/src/h2root.cxx

```diff
@@ -312,7 +312,8 @@ int main(int argc, char **argv)
 
    int lun = 10;
 #ifndef WIN32
-   hropen(lun,PASSCHAR("example"),PASSCHAR(file_in),PASSCHAR("px"),record_size,ier,7,strlen(file_in),2);
+   auto opt = PASSCHAR("px ");
+   hropen(lun,PASSCHAR("example"),PASSCHAR(file_in),opt,record_size,ier,7,strlen(file_in),2);
```

Comment:
Why increase the size of 'only' the 3rd string?  What was the result of the test?

Reply:
Fair enough.  If this is the case, we will need to add this explanation as a explanatory comment.